src/styles/_variables.scss

```diff
@@ -0,0 +1,11 @@
+
+$primary-blue: #00acdc;
+$primary-grey: #616070;
+$primary-black: #060b35;
+
+$border-color:  #f3f3f3;
+
+$side-padding : 50px;
+$card-width: 200px;
+
+$container-width: $card-width * 4 + $side-padding * 3;
```

Comment:
Better to create variable for amount of cards in a row, now it looks like magic number